atf_eregs/settings/prod.py

```diff
@@ -31,7 +31,10 @@
     HAYSTACK_CONNECTIONS['default'] = {
         'ENGINE': ('haystack.backends.elasticsearch_backend.'
                    'ElasticsearchSearchEngine'),
-        'URL': elastic_service.credentials['uri'],
+        'URL': 'http://{}:{}@{}:{}'.format(elastic_service.credentials['username'],
+                    elastic_service.credentials['password'],
+                    elastic_service.credentials['hostname'],
+                    elastic_service.credentials['port']),
```

Comment:
We should be able to change this back, pending confirmation of https://github.com/18F/kubernetes-broker/pull/29.